docs/configuration-reference.md

```diff
@@ -175,6 +178,28 @@ configs:
 [instance_mode: <string> | default = "shared"]
 ```
 
+### http_tls_config
+
+The `http_tls_config` block configures server and reading metrics via TLS, with or without certificate pinning. In 
+conjunction with the `integrations client_cert_file` and `integrations client_key_file`.  Acceptable values for
+`client_auth_type` are found in [tls.config](https://golang.org/pkg/crypto/tls/#ClientAuthType). 
+
+```yaml
+# File path to the server certificate
+[cert_file: <string>]
+
+# File path to the server key file
+[key_file: <string>]
+
+# Tells the server what is acceptable from the client. 
+# *NOTE* NoClientCert is not acceptable. If you want the server to accept any connection use `RequestClientCert`
```

Comment:
from offline discussion: this line can be removed once we stop syncing server client/client client cert settings.

Reply:
Removed/Intent change, separated out the TLS configs